recipes/cem/build.sh

```diff
@@ -0,0 +1,5 @@
+mkdir _build && cd _build
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release
+make all -j4
```

Comment:
May you use ``make all -j$CPU_COUNT`` instead of ``make all -j4`` ?

Reply:
Yes, of course. Sorry about that.

Reply:
No problem! :)